resources/tools/mdsd/devops/pipeline/stages/impl/constraintBuild/modules/Container/SetupContainer.groovy

```diff
@@ -0,0 +1,7 @@
+extendConfiguration([
```

Comment:
Can't this be moved to the `dockerized` folder and be called from a more generic constraint build module?

Reply:
Yes.